packages/appeals-service-api/src/spec/appeal-user.yaml

```diff
@@ -33,4 +33,18 @@ components:
         lpaStatus:
           type: string
           enum: ['added', 'confirmed', 'removed']
-          description: if an LPA user, the status of this user, e.g. have they logged in and confirmed their email?
+          description: if an LPA user, the status of this user, e.g. have they logged in and confirmed their email
+
+        # relations
```

Comment:
Do we return Appeals on some endpoints?
Security Token isn't anywhere is it, that should only be in Auth server?